docs/pages/auto-discovery/kubernetes.mdx

```diff
@@ -136,7 +147,38 @@ kubernetes_service:
     resources:
     - labels:
         "*": "*" # can be configured to limit the clusters to watched by this service.
+      aws:
+       # AWS role to assume when accessing EKS clusters in the AWS Account.
+       # This value is an optional AWS role ARN to assume when forwarding requests
+       # to EKS clusters.
+       assume_role_arn: arn:aws:iam::123456789012:role/iam-discovery-role
+       # External Id is an optional value that should be set when accessing
+       # your AWS account from a 3rd party access (delegate access).
+       external_id: "example-external-id"
 ```
 
+
+When configuring the `kubernetes_service.resources` parameter, the Teleport
+Kubernetes Service is set to monitor EKS clusters that are discovered by
+the Teleport Discovery Service. The monitoring process involves a label
+matching mechanism to identify and manage the EKS clusters.
+
+1. **Discovery and Label Matching**:
```

Comment:
This section is a little difficult for me to read. Would it be possible to use subheadings in place of the numbered list items and regular body text instead of the bulleted list items? Since these are full sentences, I don't think bullets make them more readable.

Reply:
I'll make a suggestion!